plugins/analytics-module-ga4/src/apis/implementations/AnalyticsApi/GoogleAnalytics4.ts

```diff
@@ -70,6 +70,7 @@ export class GoogleAnalytics4 implements AnalyticsApi {
       },
       gtagOptions: {
         debug_mode: debug,
+        send_page_view: false,
```

Comment:
if we can make this configurable, that would great!
Like how we did with debug, default value can be false but, allows the users to customize if we need it this value to be true at some point.

Reply:
Done! also tried to describe it to the readme